driver/sdk.md

```diff
@@ -1,4 +1,97 @@
 
 # Babelfish SDK
 
-* **TODO**
\ No newline at end of file
+The [Babelfish SDK](https://github.com/bblfsh/sdk/) contains tools and libraries
+required to create a Babelfish driver.
+
+## Prerequisites
+
+Babelfish has the following prerequisites:
+
+* [Docker](https://www.docker.com/get-docker)
+* [Go](https://golang.org/dl/)
+
+## Installing
+
+Install the Babelfish SDK:
+
+```bash
+$ go get github.com/bblfsh/sdk/...
+```
+
+This will fetch the sdk repository to `$GOPATH/src/github.com/bblfsh/sdk` and
+will install the `bblfsh-sdk` CLI to `$GOPATH/bin/bblfsh-sdk`.
+
+## Bootstrapping a new driver
+
+Let's say we're creating a driver for `mylang`. First step is initializing a git
+repository for the driver:
+
+```bash
+$ cd $GOPATH/src/github.com/bblfsh
+$ git init mylang-driver
+$ cd mylang-driver
+$ touch README.md
+$ git add README.md
+$ git commit -m 'initialize repository'
+```
+
+Now the driver should be bootstrapped with `bblfsh-sdk`:
```

Comment:
Also, the insertion of the git hook caught me by surprise and made me lost some time when checking out a commit prior to running bblfsh-sdk, as I could no longer commit due to missing requirements of the hook. We should explain the this here.